#151 - Fix a bug where connecting to watch from pair page ...

DreadBoy opened the pull request:
... wouldn't be remembered and app would show setup screen at next launch.

Note to reviewers: 
* Is there any other screen where user's can connect to watch?
* Is there better file where I can put key of saved preference? I decided on existing `preferences.dart` but some other file might be better.

matejdro commented:
You could add preference key into that file: https://github.com/pebble-dev/mobile-app/blob/master/lib/infrastructure/datasources/preferences.dart

Although it we might want to split it up because this file is getting pretty big

DreadBoy commented:
I decided to keep preferences backward compatible as there's no reason to change key of saved preferences. They are not consistent with other names in `datasources/preferences.dart` but whatever 🤷‍♂️.

I also moved `bootSetup` preference to same file and updated code on screens that used `bootSetup` preference.

matejdro commented on lib/ui/splash/splash_page.dart:
```diff
@@ -38,35 +38,32 @@ class _SplashPageState extends State<SplashPage> {
   @override
   Widget build(BuildContext context) {
+    final hasBeenConnected = useProvider(hasBeenConnectedProvider.last);
+    // Let's not do a timed splash screen here, it's a waste of
```
:+1:
